core/controllers/admin.py

```diff
@@ -94,6 +95,12 @@ def get(self):
             (unicode(ind), exp[0]) for ind, exp in
             enumerate(feconf.DEMO_EXPLORATIONS)]
 
+        demo_collection_ids = [
```

Comment:
I thought 'enumerate' gives a 2-tuple?

In any case, I think it's a good time to find a better way to do this demo loading thing. Can the data structure in feconf be a list of dicts, where each dict has id and source file?